### ISSUE TITLE
Enable layers option on ArcGIS Portal input broker.

### ISSUE THREAD

**Issue by davidmccorkindale** [Esri/geoportal-server-harvester#208]
It would be good if the ArcGIS Portal input broker had the option to "enable layers" the same as the ArcGIS Server input broker.

**Comment by mhogeweg** [comment]
would the goal be to harvest sub-layers of hosted feature layers as separate metadata items?

**Comment by davidmccorkindale** [comment]
Yes, the goal is to harvest sub-layers of any service as separate metadata items including the relevant metadata of a sub-layer.  This include both hosted and standard ArcGIS Server services (although the later is already supported in the AGS broker).  All our content is published to portal https://spatial.information.qld.gov.au/arcgis/home/
